pwem/protocols/protocol_export/protocol_export_emdb.py

```diff
@@ -32,19 +32,21 @@
 import pyworkflow.protocol.params as params
 from pyworkflow import VERSION_1_2
 
+from pwem.convert.atom_struct import fromPDBToCIF, fromCIFTommCIF
 from pwem.emlib.image import ImageHandler
 from pwem.convert import AtomicStructHandler, toCIF
 from pwem.protocols import EMProtocol
 from pwem.objects import FSC
 
 
-class ProtExportEMDB(EMProtocol):
-    """ generates files for volumes and FSCs to submit structures to EMDB
+class ProtExportDataBases(EMProtocol):
```

Comment:
This class name seems duplicated!!!

Reply:
We have replaced the name of the class name ProtExportEMDB by ProtExportDataBases in order to include other databases in addition to EMDB (as it was suggested by one of the referees of the article doi: 10.1021/acs.jcim.9b01032). We haven't found any other files with the old name ProtExportEMDB.